indico/modules/events/editing/controllers/backend.py

```diff
@@ -44,12 +49,44 @@ def _process(self):
 
 
 class RHEditingTags(RHEditingBase):
-    """Return all editing tags defined in the event."""
+    """Return all editing tags defined in the event or create a new tag."""
 
     def _process(self):
         return EditingTagSchema(many=True).jsonify(self.event.editing_tags)
 
 
+class RHCreateTag(RHEditingManagementBase):
```

Comment:
You need to check for unique codes here and when editing. Check e.g. `RHEquipmentTypes` (the admin one) for an example how to properly return the error in the same style webargs would report a failure so it shows up on the correct field in the client.